scripts/cipublish

```diff
@@ -10,38 +10,51 @@ function usage() {
     echo -n \
          "Usage: $(basename "$0")
 
-Publish deployment artifacts in CI.
+Publish container images to Elastic Container Registry (ECR).
 "
 }
 
-if [ "${BASH_SOURCE[0]}" = "${0}" ]
-then
-    if [ "${1:-}" = "--help" ]
-    then
-        usage
-    else
-        TAUI_CONFIG_BUCKET="s3://echo-locator-${ENV}-config-us-east-1"
-        TAUI_CONFIG_DIR="configurations/${ENV}"
-        TAUI_SITE_BUCKET="s3://echo-locator-${ENV}-site-us-east-1"
+ENVIRONMENT="${ECHOLOCATOR_ENVIRONMENT:-stgdjango}"
 
-        # Pull down variables for this environment
-        aws s3 cp "${TAUI_CONFIG_BUCKET}/taui/env.yml" \
-                  "taui/${TAUI_CONFIG_DIR}/env.yml"
-
-        echo "LOGROCKET_RELEASE: '${TRAVIS_COMMIT}'" >> "taui/${TAUI_CONFIG_DIR}/env.yml"
+if [[ -n "${GIT_COMMIT}" ]]; then
+    GIT_COMMIT="${GIT_COMMIT:0:7}"
+else
+    #GIT_COMMIT="$(git rev-parse --short HEAD)"
+    # For staging we just want latest; TODO refine
+    GIT_COMMIT="latest"
+fi
 
-        # Move the Amplify config for the proper env into the source dir
-        cp "deployment/amplify/${ENV}/src/aws-exports.js" taui/src/aws-exports.js
+function amazon_ecr_login() {
+    # Retrieves a temporary authorization token that can be used to access
+    # Amazon ECR, along with the registry URL.
```

Comment:
These comments are so helpful! Thanks for writing these in, it definitely helps this file's readability.

Reply:
Props to former ops team members!